integration-tests/tests/common/mod.rs

```diff
@@ -87,6 +89,10 @@ impl IntegrationTest {
             }
         };
 
+        let output = cmd.wait_with_output().expect("Failed to read stdout");
+        if stdout != "" {
```

Comment:
Instead of checking if the `stdout` arg is an empty string, it seems more "Rusty" to make it an `Option`. Alternatively you could use `is_empty()` on `stdout`.

Reply:
Let's just skip this check altogether. We can do an unconditional:

```rust
             assert_eq!(stdout.as_ref(), &output.stdout);
```